prow/external-plugins/needs-rebase/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/k8s-prow/git:0.1
+LABEL maintainer="baomengjiang@gmail.com"
```

Comment:
You shouldn't mark yourself as the maintainer of `needs-rebase`. I'm probably the most suitable maintainer (colew@google.com).

Reply:
Sorry for my mistake.